packages/image_picker/image_picker_ios/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -234,7 +249,11 @@ - (void)pickVideoWithSource:(nonnull FLTSourceSpecification *)source
                                              camera:[self cameraDeviceForSource:source]];
       break;
     case FLTSourceTypeGallery:
-      [self checkPhotoAuthorizationWithImagePicker:imagePickerController];
+      if (context.requestFullMetadata) {
```

Comment:
Doesn't this need the same `@available` check and fallback?

Reply:
I just realized that this change for picking a video can't be used with the current platform interface. It doesn't allow passing `requestFullMetadata` for both multi-image and videos. I can either improve the interface or remove this change for video picking (depending on what you think is better).

Reply:
I don't see any reason we wouldn't want support for this in the multi-image case, so updating that interface makes sense.

Looking at the result callback, is this change even relevant for video? It looks like the parameter is only used in a non-video codepath.

Reply:
Okay, I'll update the interface and create a PR soon. I'll also remove usage of `requestFullMetadata` for the video path, since it it isn't used, as you noticed.

Reply:
Looks like this is still missing the iOS 11 `@available` check.

Reply:
I've removed usage of `requestFullMetadata` from here since it isn't supported in the video picking.